challenges/2021_cloud_base_height/05_optuna_tuning.ipynb

```diff
@@ -0,0 +1,923 @@
+{
```

Comment:
<p>Nice summary, it would be good to have this in other notebooks.</p><p> </p><p>Given that you get the ssame amount of compute for a notebook or script on spice, why are you recommending it been converted to ascript? It might also be worth saying why you're using notebooks at all then (for better communication).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Added some explainer text